Wsl.psm1

```diff
@@ -59,12 +59,18 @@ class WslVersionInfo {
     [int]$DefaultDistroVersion
 }
 
-# Ensure IsWindows is set for Windows PowerShell to make future checks easier.
+# Provides the details of online distributions
+class WslDistributionOnline {
+    [string]$Name
+    [string]$FriendlyName
+}
+
+# Ensure IsWindowsOS is set for Windows PowerShell to make future checks easier.
 if ($PSVersionTable.PSVersion.Major -lt 6) {
-    $IsWindows = $true
+    $IsWindowsOS = $true
 }

```

Comment:
Why did you change this? It breaks the module on PowerShell 6+.

Reply:
Just saw your other comment about this. Yes, you get a warning here, but it's a false positive because the code only assigns the variable on PowerShell older than 6. The intent here is to use the built-in variable in PowerShell 6+, and just set it to true on Windows PowerShell 5.1 because we know that's always Windows. Please revert this change.

Reply:
Reverted.